package.json

```diff
@@ -180,7 +180,7 @@
   },
   "engines": {
     "npm": ">= 6.0.0",
-    "node": ">= 14.15.1"
+    "node": ">= 16.0.0"
```

Comment:
Why not the exact version we currently deploy in the dev container? v16.18.1

Reply:
My thought was, that if someone else wants to setup the project, It could be less than the 16.18.. in theory. But I will change it. Then we are safe on that side.

c21be77b5f9671e3d8d8bdb3961bb220121e01ff